sdk/parents/clientcore-parent/pom.xml

```diff
@@ -733,10 +731,19 @@
               <arg>-Xlint:-requires-transitive-automatic</arg> <!-- FIXME: this is required for now as it introduces a build failure -->
             </compilerArgs>
 
-            <!-- Default is annotation processing being off. -->
-            <proc>${compiler.proc}</proc>
-
+            <!-- Default is annotation processing being on. -->
+            <annotationProcessorPaths>
+              <annotationProcessorPath>
+                <groupId>io.clientcore</groupId>
+                <artifactId>annotation-processor</artifactId>
+                <version>1.0.0-beta.1</version> <!-- {x-version-update;io.clientcore:annotation-processor;current} -->
```

Comment:
The dependency here can't be current as this will force the annotation processor to be built before any CI can continue